src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -52,10 +52,13 @@ public Http3Connection(HttpMultiplexedConnectionContext context)
 
         _errorCodeFeature = context.ConnectionFeatures.GetRequiredFeature<IProtocolErrorCodeFeature>();
 
-        var httpLimits = context.ServiceContext.ServerOptions.Limits;
+        var serverOptions = context.ServiceContext.ServerOptions;
+        var httpLimits = serverOptions.Limits;
 
         _serverSettings.HeaderTableSize = (uint)httpLimits.Http3.HeaderTableSize;
         _serverSettings.MaxRequestHeaderFieldSectionSize = (uint)httpLimits.MaxRequestHeadersTotalSize;
+        _serverSettings.EnableWebTransport = Convert.ToUInt32(serverOptions.EnableWebTransport);
+        _serverSettings.H3Datagram = Convert.ToUInt32(serverOptions.EnableHttp3Datagrams);
```

Comment:
Is this `Convert.ToInt32` more or less efficient than just using a ternary statement? Can someone please recommend how I could double check?

Reply:
Offline feedback: remove the settings from ServerOptions, it's OK if they're always on.